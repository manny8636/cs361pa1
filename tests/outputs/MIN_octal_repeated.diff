--- outputs/MIN_octal_repeated.txt	2022-02-15 14:30:53.987061317 -0500
+++ expected/MIN_octal_repeated.txt	2021-01-25 15:27:24.000000000 -0500
@@ -0,0 +1,2 @@
+VALUE [integer]: 93
+Success!
