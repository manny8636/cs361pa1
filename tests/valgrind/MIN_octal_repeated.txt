==3062739== Memcheck, a memory error detector
==3062739== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3062739== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3062739== Command: ../json -v ../data/octal_repeated.txt
==3062739== 
Success!
==3062739== 
==3062739== HEAP SUMMARY:
==3062739==     in use at exit: 13 bytes in 1 blocks
==3062739==   total heap usage: 4 allocs, 3 frees, 16,869 bytes allocated
==3062739== 
==3062739== LEAK SUMMARY:
==3062739==    definitely lost: 13 bytes in 1 blocks
==3062739==    indirectly lost: 0 bytes in 0 blocks
==3062739==      possibly lost: 0 bytes in 0 blocks
==3062739==    still reachable: 0 bytes in 0 blocks
==3062739==         suppressed: 0 bytes in 0 blocks
==3062739== Rerun with --leak-check=full to see details of leaked memory
==3062739== 
==3062739== For lists of detected and suppressed errors, rerun with: -s
==3062739== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
