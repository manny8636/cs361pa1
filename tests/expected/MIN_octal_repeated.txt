VALUE [integer]: 93
Success!
